packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -23,7 +23,7 @@ const { prefix } = settings;
  *
  * @param {object} props props object
  * @param {string} props.heading  Heading objects
- * @param {copy} props.copy Heading copy
+ * @param {string} props.copy Heading copy
```

Comment:
@photodow I think this belongs with the `ContentBlock` pattern review. Will make note of this comment.

Reply:
I agree, but I feel like it would just be quicker to fix this here than open up a new pull request to fix it. Also, the change here doesn't appear to be part of this initial intent behind this PR to begin with. 🤷‍♂️

Reply:
ok changed the descriptions to something more accurate